graylog-storage-opensearch3/src/test/java/org/graylog/storage/opensearch3/views/export/OpenSearchExportBackendIT.java

```diff
@@ -51,10 +50,8 @@
 import static org.mockito.Mockito.mock;
 
 
-@Disabled("Temporarily disabled because of flakiness")
```

Comment:
are we sure the flakiness is gone since we didn't change any logic?

Reply:
Not at all, but since we have no better tests, I'd give it a try and see if it's still randomly failing.